client/components/reader-main/index.jsx

```diff
@@ -12,6 +12,18 @@ import React from 'react';
 import Main from 'components/main';
 import SyncReaderFollows from 'components/data/sync-reader-follows';
 
+// this exists debounced to avoid a race conditions where a ReaderMain
```

Comment:
This should get reviewed by @Automattic/reader :)

Reply:
i don't see any debouncing... ? ref-counting, yes.

Reply:
whoops, need to update the comment.
initially i tried debouncing but that doesn't solve the issue

Reply:
whole thing has been split out to: https://github.com/Automattic/wp-calypso/pull/19445